### ISSUE TITLE
Most performant prompts per task?

### ISSUE THREAD

**Issue by ryan-caesar-ramos** [google-research/FLAN#23]
If I'm understanding this correctly, in the original FLAN paper, the evaluation protocol for a single task involved averaging the accuracies achieved with the task's different prompts, with the most performant prompt being used for the test set if a dev set existed (I'm assuming this was the same with the Flan-T5 paper since I couldn't find the specifics in the paper, but maybe I just missed it).

Do the authors have records indicating what the most performant prompts were per task? Or the accuracies per prompt per task?

**Comment by shayne-longpre** [comment]
Thanks for the question Ryan. 

I think we manually experimented with different prompts on the dev sets, then chose the best ones for the test sets. For "held-in" tasks, the template will have been chosen from those documented in the training set templates.py file. For held-out tasks, unfortunately we did not get around to releasing this while I was an intern. I know that Jason posted some example prompts for MMLU CoT here: https://github.com/jasonwei20/flan-2. You may be able to remove the CoT parts of the instruction and outputs to mimic what we did. Note that we tried a few things for MMLU and there are a few design choices that helped: specifying the MMLU task/topic in the instruction, using "(A)", "(B)", etc, emulating common spacing between exemplars from training, etc..

**Comment by ryan-caesar-ramos** [comment]
Thanks! Appreciate it
